README.adoc

```diff
@@ -27,6 +27,9 @@ APK from the command line, run:
 
 Once done the .apk file can be found in the app/build/outputs/apk/ directory.
 
+== Translations
+https://github.com/Yubico/yubioath-android/blob/master/TRANSLATIONS.md[Check here] for information on how to handle translations & localisation
```

Comment:
Broken link (wrong file extension)

Reply:
It's probably better to have a relative link to the file so that it doesn't switch branches, etc. Just `link:TRANSLATIONS.adoc[Check here]` should do.
Sorry, should have caught this the first time.